R/derive_last_dose.R

```diff
@@ -1,23 +1,23 @@
-#' Derive Last Dose Date(-time)
+#' Derive Last Dose
 #'
+#' Add EX source variables from last dose to the input dataset.
 #' @param dataset Input dataset.
 #' @param dataset_ex Input EX dataset.
```

Comment:
Please add the variables expected for `dataset` and `dataset_ex`.

Reply:
Updated